app/helpers/groups_helper.rb

```diff
@@ -1,35 +1,38 @@
 module GroupsHelper
+  require 'array'
+
   def active_users(group)
-    active_memberships =
-      group.memberships.reject { |m| m.inactive? || m.archived? }
-           .sort { |a, b| a.user.lastname <=> b.user.lastname }
-    @active_users = Kaminari.paginate_array(active_memberships)
+    newbie_memberships = group.newbie_members.sort_by_name
+    active_memberships = group.active_members.sort_by_name
+
+    @active_users = Kaminari.paginate_array(newbie_memberships + active_memberships)
```

Comment:
I think this sorting method should be enabled only for group leaders and for every other user, there should be the previous sorting only by name.